### ISSUE TITLE
Infinite Circular Reference Detected

### ISSUE THREAD

**Issue by fullstackmaverick** [pb33f/wiretap#132]
Hi team,

I came across the issue and I couldn't find how could we disable that on Wiretap level.

```
wiretap version: 0.1.19 | compiled: Tue, 23 Jul 2024 17:45:30 UTC
Designed and built by Princess Beef Heavy Industries: https://pb33f.io/wiretap

 INFO  No wiretap configuration located. Using defaults
Error: infinite circular reference detected: Site: User -> SiteManager -> Site -> Organisation -> Site [255:15]
```

These are model based dependencies e.g. using relationships.

Any thoughts to resolve this would be great!

**Comment by daveshanley** [comment]
For your context: 

https://pb33f.io/libopenapi/circular-references/

**Comment by daveshanley** [comment]
However, I just realized there is no way to disable it in wiretap.

**Comment by daveshanley** [comment]
Fixed in `v0.1.20` 

If the model builds, but non-breaking issues like circular references are returned, then `wiretap` will spit out warnings, but continue on as normal. 

![Screenshot 2024-08-08 at 9 25 38 AM](https://github.com/user-attachments/assets/50fe7602-11e8-4cb2-a50c-7766b1c3835e)

**Comment by fullstackmaverick** [comment]
Thanks for the fix! Amazing support.
